src/models/validationModels/lineValidationModel.ts

```diff
@@ -0,0 +1,16 @@
+import Constants from '~/constants/constants';
```

Comment:
Constants is a variable and should be camel case, but I see that we have earlier written it using pascal case..?

Reply:
To be fixed in #822